### DISCUSSION TITLE
Add custom config Caddy to the static binary

### DISCUSSION THREAD

**Discussion by JuanLuisGarciaBorrego** [php/frankenphp#1239] [Q&A]
Hello franken's!

**Is it possible to the configure caddyfile in static binary?**
For example, change custom port:
:{_custom_port_}{
    ...rest config
}

I have seen that it is possible to add extensions[ in this issue ](https://github.com/dunglas/frankenphp/pull/1210#issue-2695160125)

**What I'm trying to do is:**
I have a server with traditional caddy setup for my frontend application.

When I try to run the binary 
` sudo ./my-app php-server --domain api.domain.com`

get error:
_Error: loading new config: http app module: start: listening on :443: listen tcp :443: bind: address already in use_

Any ideas /alternative?

Thanks!!

**Comment by AlliBalliBaba** [comment]
I think you can just add a `Caddyfile` to the root of your project and it will be used by default when running `./my-app php-server`
